com.unity.render-pipelines.universal/Runtime/UniversalRenderPipelineCore.cs

```diff
@@ -284,7 +285,8 @@ public struct PostProcessingData
     public static class ShaderKeywordStrings
     {
         public static readonly string MainLightShadows = "_MAIN_LIGHT_SHADOWS";
-        public static readonly string MainLightShadowCascades = "_MAIN_LIGHT_SHADOWS_CASCADE";
+        public static readonly string MainLightShadowsCascades = "_MAIN_LIGHT_SHADOWS_CASCADE";
```

Comment:
This is public API breakage, unfortunately

Reply:
I just want to match name convention, it was just my thoughts. breaking public API is needed to be careful. let me restore it as before